apps/question-the-docs/backend/document/routes.py

```diff
@@ -1,25 +1,20 @@
-import numpy as np
-import pandas as pd
 from fastapi import APIRouter, Body, Request
-from lancedb.context import contextualize
 from superduperdb import superduper
-from superduperdb.container.document import Document as D
-from superduperdb.container.listener import Listener
-from superduperdb.container.vector_index import VectorIndex
 from superduperdb.db.mongodb.query import Collection
-from superduperdb.ext.openai.model import OpenAIChatCompletion, OpenAIEmbedding
 
 from backend.document.models import Query
+from backend.config import settings
 
 document_router = APIRouter(prefix="/document", tags=["docs"])
 
 
-def concept_assist_prompt_build(famous_person, query, context):
+def concept_assist_prompt_build(famous_person):
     return (
-        "Following is a Query and some context around the query, please "
-        "try to give an answer for the query with the given context and "
-        f"please respond in the voice of {famous_person}\nQuery: "
-        f"{query}\nContext: {context}"
+        f'Use the following description and code-snippets aboout SuperDuperDB to answer this question about SuperDuperDB in the voice of {famous_person}\n'
+        'Do not use any other information you might have learned about other python packages\n'
+        'Only base your answer on the code-snippets retrieved\n'
```

Comment:
shouldn't it generalise with code snippet and general concept?
so it should be 

```python 
Only base your answer on the Context: {context}
```

Reply:
@blythed

Reply:
No, that's not my experience. However, we can experiment in the next step.